src/fields2cover/swath_generator/swath_generator_base.cpp

```diff
@@ -48,12 +49,45 @@ F2CSwaths SwathGeneratorBase::generateSwaths(double angle,
   F2CPoint min_point(rot_poly.getDimMinX(), rot_poly.getDimMinY());
   auto seed_curve = rot_poly.createStraightLongLine(min_point, 0.0);
 
-  double curve_y {-0.5 * op_width};
   F2CMultiLineString paths;
-  while (field_height > curve_y + (allow_overlap ? 0.0 : 0.5 * op_width)) {
-    curve_y += op_width;
+
+  double increment = op_width;
```

Comment:
Is there a reason for using increment instead of op_width? And for putting this line here instead of line 81?